presto-hive-hadoop2/bin/run_hive_abfs_tests.sh

```diff
@@ -8,7 +8,11 @@ test -v ABFS_CONTAINER
 test -v ABFS_ACCOUNT
 test -v ABFS_ACCESS_KEY
 
-start_docker_containers
+cleanup_hadoop_docker_containers
+start_hadoop_docker_containers
+
+test_directory="$(date '+%Y%m%d-%H%M%S')-$(uuidgen | sha1sum | cut -b 1-6)"
```

Comment:
Why hash and cut the UUID? Using the full UUID doesn't make it any easier or harder to read, and it's not so many characters that it should cause problems.

I'd suggest a slightly more verbose name, in case someone needs to find a particular test at any point, something like `"test-$(date +%Y-%m-%dT%H.%M.%S)-id-$(uuidgen)"`.

Reply:
This came from @findepi in e364b6b493fb60734a11332e727eae91b829b707 for S3. I'm inclined to agree, except for adding dots in the name.

Reply:
The `sha1sum` part might be an attempt to normalize the output case as `uuidgen` on macOS is uppercase but Linux is lowercase.

Reply:
The case shouldn't matter, because it's still only generated once (in any case, that's what `tr '[:upper:]' '[:lower:]'` is for).

I wasn't sure about the dots, either; dashes work just as well.

Reply:
> This came from @findepi in e364b6b for S3

`sha1sum` is necessary if we want to `cut` (uuid's prefix is not random).

`cut` can be used because this is only for collision ties (we have date/time as well), so full uuid is enterpreisely redundant